scripts/emulator-testing/emulators/database-emulator.ts

```diff
@@ -26,11 +26,11 @@ export class DatabaseEmulator extends Emulator {
 
   constructor(port = 8088, namespace = 'test-emulator') {
     super(
-      'firebase-database-emulator-v4.9.0.jar',
+      'firebase-database-emulator-v4.11.0.jar',
```

Comment:
Can you extract this version out to a template?

Reply:
Done.